txpipe/plotting/correlations.py

```diff
@@ -374,11 +498,21 @@ def make_theory_plot_data(data, cosmo, obs, label, smooth=True, xi=None):
             # Optionally also compute correlation functions
             if xi:
                 theta, *_ = obs[(GAMMA, i, j)]
-                try:
```

Comment:
I think the except version is better here.  Looks like it's commented here - do you still get the double free in the newest ccl?